app/controllers/dashboard_controller.rb

```diff
@@ -15,6 +15,7 @@ def index
     set_admin_courses_if_admin
     @pres = DashboardPresenter.new(current_courses, past_courses,
                                    @submitted, @strictly_current, current_user)
+    @instructor_view = params.key?(:teach) and user_signed_in?
```

Comment:
I think the better way to handle this is inside the DashboardPresenter, by passing the 'teach' parameter to the presenter.

Using a separate instance variable makes it more complicated to get all the correct behaviors (such as showing the link to instructor orientation but not the create course button, if the user has not already completed that training module). With the presenter approach, I think there will be no changes needed in the view templates.

Reply:
Okay, I will make the required changes. Thank you for the review